src/seed/stub/ts.stub

```diff
@@ -0,0 +1,14 @@
+import * as Knex from "knex";
+
+exports.seed = function (knex: Knex): Promise<any> {
+    // Deletes ALL existing entries
+    return knex("table_name").del()
+        .then(function () {
+            return Promise.all([
```

Comment:
I updated this branch for the TypeScript stub, have a patch ready for the other stubs once you find it good to merge.